tests/test_server.c

```diff
@@ -2490,6 +2780,40 @@ void TestRaft_candidate_recv_appendentries_from_same_term_results_in_step_down(
 
     raft_set_current_term(r, 1);
     raft_become_candidate(r);
+    CuAssertTrue(tc, raft_is_candidate(r));
+    CuAssertIntEquals(tc, -1, raft_get_voted_for(r));
+
+    memset(&ae, 0, sizeof(msg_appendentries_t));
+    ae.term = 2;
```

Comment:
what is the term now, is it still 1 (because raft_become_prevoted_candidate has not occurred)? Shold the AE message specify the same term 1?

Reply:
Indeed, will fix.